plugin-core/src/main/java/appland/webviews/findings/FindingsOverviewEditor.java

```diff
@@ -53,37 +51,6 @@ protected void setupInitMessage(@Nullable List<ScannerFinding> initData, @NotNul
 
     @Override
     protected void afterInit(@Nullable List<ScannerFinding> initData) {
-        assert initData != null;
-
-        var deduplicatedFindings = deduplicateFindingsByHash(initData);
-        var ruleIds = findUniqueRuleIds(initData);
-        var impactDomainsLowercase = findUniqueImpactDomainsLowercase(initData);
-        var impactDomainCounts = createImpactDomainCounts(initData);
-        var uniqueImpactDomainCounts = createImpactDomainCounts(deduplicatedFindings);
-
-        TelemetryService.getInstance().sendEvent("analysis:view_overview", event -> {
-            // properties
-            event.property("appmap.analysis.rules", StringUtil.join(ruleIds, ","));
-            event.property("appmap.analysis.impact_domains", StringUtil.join(impactDomainsLowercase, ","));
-
-            // metrics
-            event.metric("num_findings.num_total_findings", (double) initData.size());
-            event.metric("num_findings.num_unique_findings", (double) deduplicatedFindings.size());
-
-            impactDomainCounts.forEach((domain, count) -> {
-                if (count > 0) {
-                    event.metric("num_findings.num_" + domain.getJsonId().toLowerCase(), count.doubleValue());
-                }
-            });
-
-            uniqueImpactDomainCounts.forEach((domain, count) -> {
-                if (count > 0) {
-                    event.metric("num_findings.num_unique_" + domain.getJsonId().toLowerCase(), count.doubleValue());
-                }
-            });
-
-            return event;
-        });
     }
```

Comment:
It looks like this is now an empty method, should we just remove the method entirely?